coalib/processes/BearRunning.py

```diff
@@ -5,8 +5,8 @@
 from coalib.bears.GlobalBear import GlobalBear
 from coalib.bears.LocalBear import LocalBear
 from coalib.misc import Constants
-from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
 from coalib.processes.CONTROL_ELEMENT import CONTROL_ELEMENT
+from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
```

Comment:
I think `LogMessage` doesn't hurt at the moment. Its a good abstraction while passing log messages as it contains message along with its severity. And it also saves us passing an extra parameter.
However, this is not really consistent, as many places has got 2 params - message and log level instead of directly using `LogMessage`.